mscore/palette.cpp

```diff
@@ -354,8 +360,105 @@ void Palette::mouseDoubleClickEvent(QMouseEvent* ev)
                   Measure* m = sel.startSegment()->measure();
                   QRectF r = m->staffabbox(sel.staffStart());
                   QPointF pt(r.x() + r.width() * .5, r.y() + r.height() * .5);
+                  pt += m->system()->page()->pos();
```

Comment:
This was often failing before for page > 1, as pt is now relative to the page rather than to the start of the score.  Not sure if there is a more elegant way to handle this.